mkdocs.yml

```
@@ -26,6 +26,7 @@ nav:
     - Browser/Acceptance Testing: browser-acceptance-tests.md
     - JavaScript Unit Testing: javascript-unit-tests.md
     - Python Unit Testing: python-unit-tests.md
+    - Functional Browser Testing: functional-browser-testing.md
```

```suggestion
    - Functional Testing: functional-testing.md
```

I want to avoid the phrase "browser" testing here, because it could be confused with cross-browser testing.